src/main/java/com/kodilla/ecommercee/controller/OrderController.java

```diff
@@ -33,12 +33,12 @@ public OrderDto getOrder(@RequestParam("orderId") Long orderId) {
 
     @PostMapping(value = "createOrder", consumes = APPLICATION_JSON_VALUE)
     public void createOrder(@RequestBody OrderDto orderDto) {
-        orderRepository.save(new Order(orderDto.getName()));
+        orderRepository.save(new Order());
```

Comment:
nie rozumiem co autor miał na myśli tutak :)

Reply:
No właśnie na razie jest to "zaślepka", bo osoba tworząca OrderDto, serwis, mapper i repository będzie musiała to uzupełnić. :-) Ja po prostu skupiłem się na samej encji Order, bo tak brzmi task w jirze.

Reply:
Lepiej było chyba nic tutaj po prostu nie wpisywać skoro to ma być tylko "wstępna implementacja"...